### ISSUE TITLE
Privacy manifest missing

### ISSUE THREAD

**Issue by jrogccom** [NielsenDigitalSDK/nielsenappsdk-ios#2]
Apple's new regulations require that third party libraries have a [privacy manifest](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).  Not having it can cause dependent projects' builds to be rejected as invalid.

**Comment by AmerigoM** [comment]
The is an [issue](https://github.com/NielsenDigitalSDK/nielsenappsdk-ios-dynamic-spm-global/issues/3) open in the NielsenSDK-XC repo. However I would like to know if they plan to fix the issue for the NielsenAppSDK too.

@akashnielsen @digappsdkiOS @mpetrovnielsen @yeolerajesh @jaggsu01 could you give us an update on this matter?

**Comment by akashnielsen** [comment]
Yes @AmerigoM changes will be done across all repos but it may take time until this Friday.  If you folks are blocked, feel free to use the TD [links](https://github.com/NielsenDigitalSDK/nielsenappsdk-ios-dynamic-spm-global/issues/3#issuecomment-2096672243) and get your apps live in the store.

**Comment by AmerigoM** [comment]
I updated Nielsen to the new version 9.3.0.0 and the problem was solved. This issue can be closed. Thanks.
